pkg/api/api.go

```diff
@@ -527,6 +527,9 @@ func (hs *HTTPServer) registerRoutes() {
 		// metrics
 		// DataSource w/ expressions
 		apiRoute.Post("/ds/query", requestmeta.SetSLOGroup(requestmeta.SLOGroupHighSlow), authorize(ac.EvalPermission(datasources.ActionQuery)), hs.getDSQueryEndpoint())
+		if hs.Features.IsEnabledGlobally(featuremgmt.FlagDsQueryConvert) {
+			apiRoute.Post("/ds/query/convert", authorize(ac.EvalPermission(datasources.ActionQuery)), hs.getDSQueryConvertEndpoint())
```

Comment:
we do not want anything new on /api -- lets try exposing this on /apis

Reply:
my plan was to add both. I am happy to skip the `/api` version if we can directly use the `/apis` one, can you point me to an example of how to add a new endpoint there?

Reply:
if subresource makes sense:
https://github.com/grafana/grafana/blob/main/pkg/registry/apis/datasource/register.go#L217

it we want it one layer up (eg, not scoped to a specific instance) there is not yet a great way, but will be pretty straightforward with the work todd is doing now for plugins+aggregation

Reply:
added both, this function now simply calls the logic defined for `/apis` but allows us to roll this out independently from the rest of `/apis`.